pyflagser/flagser.py

```diff
@@ -51,84 +161,86 @@ def flagser(flag_matrix, min_dimension=0, max_dimension=np.inf, directed=True,
         :math:`\\mathbb{F}_p = \\{ 0, \\ldots, p - 1 \\}` where
         :math:`p` equals `coeff`.
 
-    approximation : int, optional, default: ``-1``
+    approximation : int or None, optional, default: ``None``
         Skip all cells creating columns in the reduction matrix with more than
         this number of entries. Use this for hard problems; a good value is
         often ``100,000``. Increase for higher precision, decrease for faster
-        computation. A negative value computes highest possible precision.
+        computation. If ``None``, no approximation is made and all cells are
+        used. For more details, please refer to [1]_.
 
     Returns
     -------
     out : dict of list
-        A dictionary holding the results of the flagser computation. Each
-        value is a list of length `max_dimension` - `min_dimension` + 1. The
-        key-value pairs in `out` are as follows:
+        A dictionary with the following key-value pairs:
 
         - ``'dgms'``: list of ndarray of shape ``(n_pairs, 2)``
           A list of persistence diagrams, one for each dimension greater
           than or equal than `min_dimension` and less than `max_dimension`.
           Each diagram is an ndarray of size (n_pairs, 2) with the first
           column representing the birth time and the second column
           representing the death time of each pair.
+        - ``'betti'``: list of int
+          Betti number at filtration value `max_edge_length` per dimension
+          greater than or equal than `min_dimension` and less than
+          `max_dimension`.
         - ``'cell_count'``: list of int
           Cell count per dimension greater than or equal than
```

Comment:
How about "Cell count (number of simplices) at filtration value `max_edge_length` per dimension [...]"?